app-constants.js

```diff
@@ -40,7 +41,6 @@ const kEnvironmentVariables = [
   "DATABASE_URL",
   "PAGE_TOKEN_TIMER",
   "SENTRY_DSN",
-  "SERVER_URL",
```

Comment:
this was a dupe of line 10.

Reply:
Did a quick search+filter and looks like this was the only duped key, good catch!